### DISCUSSION TITLE
what is ToDo

### DISCUSSION THREAD

**Discussion by TrapinchO** [DraqueT/PolyGlot#1085] [Q&A] [answered]
I have seen classes for "ToDo" of some kind  but haven't seen it mentioned in the documentation or (iirc) in the program. Is it something internal or an unifinished feature?

**Comment by TrapinchO** [comment, marked as answer]
no answer yet

**Reply by DraqueT** [reply]
Ah sorry for missing this before! The TODO classes actually represent the TODO menu that is on the right side of PolyGlot. To add entries to it, just right click in the field.

**Reply by TrapinchO** [reply]
thats neat!
I will make issue for documentation
